src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -73,6 +73,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             End Get
         End Property
 
+        Friend ReadOnly Property GetScannerOptions As VisualBasicParseOptions
```

Comment:
@cston I'll do another review of the usages, as I think all of the feature utils usages can go through this method. If so, I'll rename it `Options` afterwards.